textattack/shared/attacked_text.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, text_input, attack_attrs=None):
             )
         # Format text inputs.
         self._text_input = OrderedDict([(k, v) for k, v in self._text_input.items()])
-        self.words = words_from_text(self.text)
+        self.words, self.words2char_offset = words_from_text(self.text, words2char=True)
```

Comment:
If you keep this, can you rename it, and add a clarifying comment?

possible better name: `word_positions_in_text`, maybe?

Reply:
Done